dan_lombardy/data/README

```diff
@@ -0,0 +1 @@
+PLACEHOLDER so /data file will show up in git 
```

Comment:
This shouldn't be necessary. If you're having troubles creating the file because there's no directory, check for and create a directory first.